ci/pipelines/deploy-to-test.yml

```diff
@@ -2770,6 +2780,61 @@ jobs:
         params:
           image: cardid-ecr-registry-test/image.tar
           additional_tags: cardid-ecr-registry-test/tag
+          
+  - name: deploy-webhooks
+    serial: true
+    serial_groups: [deploy-application]
+    plan:
+      - get: webhooks-ecr-registry-test
+        trigger: true
+      - get: nginx-proxy-ecr-registry-test
+        trigger: true
+      - get: telegraf-ecr-registry-test
+        trigger: true
+      - get: pay-infra
+      - get: pay-ci
+      - load_var: application_image_tag
+        file: webhooks-ecr-registry-test/tag
```

Comment:
I think this file won't exist without the equivalent `parse-release-tag` job - you might have to hardcode or do some wrangling

Reply:
pretty sure this doesn't rely on any other job (or step), it's just relying on the file created by the resource, as documented here https://github.com/concourse/registry-image-resource#files-created-by-the-resource

Reply:
Ah you're right. What sort of tag do we get there (i.e. at some stage will we have to do some formatting to get a nice `123-release` for use further down the pipeline)?

Reply:
yep, the resource dictates which tags it will trigger on here https://github.com/alphagov/pay-ci/blob/867aa56d7acaf262af50b1ff56059f2e0c3ca6d9/ci/pipelines/deploy-to-test.yml#L273

At the moment I have it listening for test_release-XXX so we can deploy stuff like that without polluting the `release-XXX` space, which might get confusing given downgrade protection etc. So at the moment this will deploy any tag that is pushed to ecr webhooks of form `test_release-XXX`